data/korath/korath weapons.txt

```diff
@@ -1009,3 +1009,32 @@ effect "firelight ring"
 	"random angle" 360
 	"random velocity" 2
 	"velocity scale" 0.1
+
+outfit "Microbot Factory"
```

Comment:
Should this be a secondary weapon? I'm a bit worried about AI cycling it constantly while something is in turret range with the hull healing on it, but still can't test myself atm to know for sure if it does or not.

Reply:
Hmm, I'm not so sure how that works, if there's no side effect of making it secondary I'll can just do that

Reply:
If anything this should be *producing* ammo, not consuming it.  Not a secondary.